middleware/handlers.go

```diff
@@ -51,43 +51,43 @@ func createConnection() *sql.DB {
 	return db
 }
 
-// CreateUser create a user in the postgres db
-func CreateUser(w http.ResponseWriter, r *http.Request) {
+// CreateJob create a job in the postgres db
+func CreateJob(w http.ResponseWriter, r *http.Request) {
 	// set the header to content type x-www-form-urlencoded
 	// Allow all origin to handle cors issue
 	w.Header().Set("Context-Type", "application/x-www-form-urlencoded")
 	w.Header().Set("Access-Control-Allow-Origin", "*")
 	w.Header().Set("Access-Control-Allow-Methods", "POST")
 	w.Header().Set("Access-Control-Allow-Headers", "Content-Type")
```

Comment:
you could use this middleware to avoid needing to write all of this headers for all the handlers you have: https://github.com/rs/cors